.github/workflows/inspect.yml

```
@@ -0,0 +1,47 @@
+      matrix:
+        target:
+          - { prettyname: Cross Platform, triplet: x86_64-unknown-linux-gnu, cwd: '.' }
+          - { prettyname: RISC-V64, triplet: riscv64gc-unknown-none-elf, cwd: 'kernel' }
+          - { prettyname: LoongArch64, triplet: loongarch64-unknown-none, cwd: 'kernel' }
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Rust
+        uses: ./.github/workflows/actions/setup-rust
```

actionlint (1.7.7)

description is required in metadata of "Setup Rust" action at "/home/jailuser/git/.github/workflows/actions/setup-rust/action.yml"

(action)